src/phoenix/server/api/types/Dimension.py

```diff
@@ -43,6 +44,7 @@ def drift_metric(
         info: Info[Context, None],
         metric: ScalarDriftMetric,
         time_range: Optional[TimeRange] = None,
+        dataset_role: Optional[DatasetRole] = None,
```

Comment:
does the default types not work here? https://strawberry.rocks/docs/types/input-types

Reply:
don't think so, we had to something similar [here](https://github.com/Arize-ai/phoenix/blob/1a411107bbb8c03e7de6cde84dd8a39101fa15d7/src/phoenix/server/api/types/EmbeddingDimension.py#LL216C24-L216C44). the default values doesn't do anything for static typing (or even runtime typing)

Reply:
never mind. i got mixed up. will update